assets/app/scripts/extensions/javaLink.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 angular
-  .module('openshiftConsoleExtensions')
+  .module('openshiftConsoleExtensions', ['openshiftConsole'])
```

Comment:
I thought about that & initially had `extensions/consoleExtensions.js` file & just these two lines.  I think its good to declare the mod apart from one of its uses.
Shall I put it back?